src/Activities/Internal/ActivityMetricsSender.cs

```diff
@@ -18,17 +21,23 @@ internal sealed class ActivityMetricsSender : IActivitiesEventSender, IDisposabl
 		private readonly Meter m_meter;
 		private readonly Counter<double> m_activityCounter;
 		private readonly Counter<double> m_healthCheckActivityCounter;
+		private readonly Histogram<double> m_activityHistogram;
+		private readonly Histogram<double> m_healthCheckActivityHistogram;
 		private readonly IExecutionContext m_context;
 		private readonly IHostEnvironment m_hostEnvironment;
+		private readonly IOptions<MonitoringOption> m_monitoringOption;
 		private readonly ArrayPool<KeyValuePair<string, object?>> m_arrayPool;
 
-		public ActivityMetricsSender(IExecutionContext executionContext, IHostEnvironment hostEnvironment)
+		public ActivityMetricsSender(IExecutionContext executionContext, IHostEnvironment hostEnvironment, IOptions<MonitoringOption> monitoringOption)
 		{
 			m_context = executionContext;
 			m_hostEnvironment = hostEnvironment;
+			m_monitoringOption = monitoringOption;
 			m_meter = new Meter("Microsoft.Omex.Activities", "1.0.0");
 			m_activityCounter = m_meter.CreateCounter<double>("Activities");
 			m_healthCheckActivityCounter = m_meter.CreateCounter<double>("HealthCheckActivities");
+			m_activityHistogram = m_meter.CreateHistogram<double>("Activities");
+			m_healthCheckActivityHistogram = m_meter.CreateHistogram<double>("HealthCheckActivities");
 			m_arrayPool = ArrayPool<KeyValuePair<string, object?>>.Create();
```

Comment:
I may be mistaken, but I would avoid calling method in the constructor. They may throw exception or implement complex logic, and the constructor is not the best place for that.
Feel free to ignore this comment if that's not the case, or if you want to address this issue in another PR.

Reply:
It is safe to do so as this basically creates a new object and doesn't change any data. 
I also want to keep them here because Counter/Histogram should not be newly created everytime a request come as it is expensive to do